fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/InviteFragment.kt

```diff
@@ -59,6 +63,8 @@ class InviteFragment : Fragment() {
     }
 
     handleBackNav()
+    setupCopyServerButton(binding)
+    setupCopyIdentifierButton(binding)
```

Comment:
I'd save the binding as a class attribute to avoid passing it around (it's just a preference but also how it's usually done in the codebase)